<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBoxDefaultOutputFont">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonOutputFont">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QLabel" name="labelPredefLang">
     <property name="text">
      <string>Predefined language definitions:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBoxUpdateCheck">
     <property name="text">
      <string>Automatically check for new program versions</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelOutputFont">
     <property name="text">
      <string>Output pane font:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelDataLocation">
     <property name="text">
      <string>Language data locations:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidgetAdditionalLang">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Filename prefix</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Native name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxDataLocation">
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>System-wide paths</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>User paths</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="3">
    <widget class="QLabel" name="labelAdditionalLang">
     <property name="text">
      <string>Additional language definitions:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidgetPredefLang">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Filename prefix</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Native name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelTessdataLocation">
     <property name="text">
      <string>Language definitions path:</string>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBoxEncoding">
     <item>
      <property name="text">
       <string>System encoding</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UTF-8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="16" column="0" colspan="3">
    <widget class="QWidget" name="widgetAddRemoveLang" native="true">
     <layout class="QHBoxLayout" name="horizontalLayoutAddRemoveLang">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonAddLang">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset theme="list-add">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRemoveLang">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset theme="list-remove">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBoxDictInstall">
     <property name="text">
      <string>Query to install missing spellcheck dictionaries</string>
     </property>
    </widget>
   </item>
   <item row="20" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelEncoding">
     <property name="text">
      <string>Text file encoding:</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="3">
    <widget class="QWidget" name="widgetAddLang" native="true">
     <layout class="QHBoxLayout" name="horizontalLayoutAddLang">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditLangPrefix">
        <property name="toolTip">
         <string>The prefix of the tesseract language data file</string>
        </property>
        <property name="placeholderText">
         <string>Prefix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLangCode">
        <property name="toolTip">
         <string>The ISO 639-1 language code followed by the country code</string>
        </property>
        <property name="placeholderText">
         <string>ISO Code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLangName">
        <property name="toolTip">
         <string>The name of the language</string>
        </property>
        <property name="placeholderText">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddLangOk">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-ok">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddLangCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-cancel">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelSpellLocation">
     <property name="text">
      <string>Spelling dictionaries path:</string>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditTessdataLocation">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditSpellLocation">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
